src/test/java/ScattTest.java

```diff
@@ -9,13 +13,68 @@
  * @author Erik Cole
  */
 public class ScattTest {
+    /**
+     * Get a string representing a path to a project test resource.
+     * Project test resources should be put in src/test/resources.
+     * @param resName The name of the resource file
+     * @return String representation of path to resource
+     */
+    private String getTestResource(String resName) {
```

Comment:
Thanks for making this guy. I can tell that it will (and has) cut down on a lot of code duplication.